package ch04network.sub4trywithresource.autocloseable;

public class CallException extends Exception {

    public CallException(String message) {
        super(message);
    }
}
